src/utils/getContributedRepos.ts

```diff
@@ -0,0 +1,24 @@
+interface PRResponse {
+    data: [{
+        full_name: string;
+    }]
+    meta: {
+        itemCount: number;
+    };
+}
+
+export const countUniqueRepos = (response: PRResponse | null) => {
+    if (!response?.data) {
+        return 0;
+    }
+
+    const { data } = response;
+    const uniqueRepos = (new Set<string>);
+
+    // eslint-disable-next-line no-loops/no-loops
```

Comment:
Instead of disabling this eslint rule try using .map or other functional methods.

Reply:
ok, i'll see what i can do.